tools/code-owners-parser/CodeOwnersManualTester/CodeOwnersManualTester.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
```

Comment:
Did you consider making them [errors](https://stackoverflow.com/a/62116924/986533) ?

Reply:
Sure, why not. Nothing in the ad-hoc project is using them but they'll be consistent with the rest of the projects in the solution.

Reply:
I'm going to be removing this project altogether. You have something similar already in the solution.